docs/components/concepts/user-task-listeners.md

```diff
@@ -186,16 +186,12 @@ client.newWorker()
 
 #### On correcting the assignee
 
-The assignee can be corrected in the `creating` listener if and only if the process hasn't specified an assignee for this user task already.
-For example, if the user task's `assignee` expression evaluates to `null`.
+The assignee can be corrected in the `creating` listener if the process hasn't specified an assignee for this user task already. For example, if the user task's `assignee` expression evaluates to `null`.
```

Comment:
>if and only if

This is a common way to describe concepts in programming, where the rule is explicitly strict. Users can only use the `creating` listener to correct the assignee if and only if that condition holds. Is there a reason to change this to "only if"? @christinaausley

Reply:
Exactly. I was wondering whether it's currently clear that you cannot correct the assignee in the `creating` listener if the process has specified an assignee for this user task already. In that case, the request to correct the assignee is rejected.